rq/cli/cli.py

```diff
@@ -217,6 +220,7 @@ def worker(cli_config, burst, logging_level, name, results_ttl,
                                          connection=cli_config.connection,
                                          default_worker_ttl=worker_ttl,
                                          default_result_ttl=results_ttl,
+                                         default_job_monitoring_interval=job_monitoring_interval,
```

Comment:
Could you please change the keyword `default_job_monitoring_interval` to `job_monitoring_interval`?

Reply:
Done.